backend/assignment-1-cezaram28/Assignment 1/src/main/java/com/cezaram28/Assignment1/controller/VotesController.java

```diff
@@ -0,0 +1,45 @@
+package com.cezaram28.Assignment1.controller;
+
+import com.cezaram28.Assignment1.dto.AnswerDTO;
+import com.cezaram28.Assignment1.dto.QuestionDTO;
+import com.cezaram28.Assignment1.dto.UserDTO;
+import com.cezaram28.Assignment1.service.UserManagementService;
+import com.cezaram28.Assignment1.service.VoteManagementService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class VotesController {
+    private final VoteManagementService voteService;
+    private final UserManagementService userService;
+
+    @PostMapping("/votes/question/upvote/{userId}")
```

Comment:
Again, you could get the user ID from the security context. Instead, you should get the question ID from the path (like: `questions/{questionId}/votes`).